book/modules/appendix1-exercises.tex

```diff
@@ -52,59 +94,253 @@
 				If so, how many? Justify your answer.
 		\end{enumerate}
 
-		\prob On Kokoro's farm, there is a cage with $35$ animals, some of which 
-		are chickens
-		and some of which are rabbits. Kokoro counted the total number of legs in the
-		cage and found that there were $94$ legs in all (notably, each chicken
-		has exactly two legs and each rabbit has four legs). Kokoro decides
-		to use this information to figure out how many chickens and how many
-		rabbits there are\footnote{ This problem based on a
-		classical Chinese problem from the ancient Chinese treatise
-		\emph{Mathematical Classic of Master Sun} (or \emph{Sunzi Suanjing}) written
-		during 3rd to 5th centuries \textsc{a.d.}}.
+		\begin{solution}
+			\begin{enumerate}
+				\item If $\vec b=\mat{7 \\ 14}$, then the vector equation
+					becomes
+					\[
+						x\mat{2 \\ 4}+y\mat{8 \\ 16}=\mat{7 \\ 14}.
+					\]
+					Convert it to a system of linear equations:
+					\[
+						\systeme{2x+8y=7,4x+16y=14}.
+					\]
+					Row reduce this system, and we get
+					\[
+						\systeme{x+4y=3.5, 0x+0y=0}.
+					\]
+					The solution to this system is then
+					\[
+						\left\{
+						\begin{array}
+							{ccc} x & = & 3.5-4t \\ y & = & t
+						\end{array}\right. (t\in \mathbb{R}).
+					\]
+					This system is consistent.
+
+				\item There are vectors $\vec{b}$ that makes the system consistent.
+					For instance, $\vec b=\mat{5 \\ 10}$ is such a vector. In
+					general, any vector $\vec b$ with $\vec b=\mat{t \\ 2t}$ for
```

Comment:
-> `any vector of the form $\vec b = ...$ where $t\in \R$ makes the system consistent`